admin/installer/_images.py

```diff
@@ -163,14 +147,14 @@ class PackerConfigure(PClass):
         ubuntu-14.04 is the only one implemented so far.
     :ivar configuration_directory: The directory containing prototype
         configuration templates.
-    :ivar source_ami: The AMI ID to use as the base image.
+    :ivar source_ami_map: The AMI map containing base images.
     """
     build_region = field(type=RegionConstant, mandatory=True)
     publish_regions = pvector_field(item_type=RegionConstant)
     template = field(type=unicode, mandatory=True)
     distribution = field(type=unicode, mandatory=True)
     configuration_directory = field(type=FilePath, initial=PACKER_TEMPLATE_DIR)
-    source_ami = field(type=unicode, mandatory=True)
+    source_ami_map = pmap_field(key_type=RegionConstant, value_type=unicode)
```

Comment:
No. Confusingly the `*_field` functions produce mandatory fields by default.
 * https://pyrsistent.readthedocs.org/en/latest/api.html#pyrsistent.pmap_field